src/Akka.Persistence.Azure/Snapshot/AzureBlobSnapshotStoreSettings.cs

```diff
@@ -61,11 +66,20 @@ public sealed class AzureBlobSnapshotStoreSettings
         public static AzureBlobSnapshotStoreSettings Create(Config config)
         {
             var connectionString = config.GetString("connection-string");
-            var containerName = config.GetString("container-name");
             var connectTimeout = config.GetTimeSpan("connect-timeout", TimeSpan.FromSeconds(3));
             var requestTimeout = config.GetTimeSpan("request-timeout", TimeSpan.FromSeconds(3));
             var verbose = config.GetBoolean("verbose-logging", false);
-            return new AzureBlobSnapshotStoreSettings(connectionString, containerName, connectTimeout, requestTimeout,
+            var containerName = config.GetString("container-name", DefaultContainerName);
```

Comment:
Same as above about loading name in runtime.

Reply:
These values should be stored in HOCON.